components/dm/spec/topology_dm.go

```diff
@@ -86,9 +86,9 @@ type (
 
 	// DMTopologySpecification represents the specification of topology.yaml
 	DMTopologySpecification struct {
-		GlobalOptions GlobalOptions `yaml:"global,omitempty"`
-		// MonitoredOptions MonitoredOptions   `yaml:"monitored,omitempty"`
-		ServerConfigs DMServerConfigs    `yaml:"server_configs,omitempty"`
+		GlobalOptions GlobalOptions `yaml:"global,omitempty" validate:"global:editable"`
+		// MonitoredOptions MonitoredOptions   `yaml:"monitored,omitempty" validate:"monitored:editable"`
```

Comment:
Don't we need the monitor options?

Reply:
dm-ansible will not deploy exporter, I will add it if we do want this.
Also, I think it's not a good idea to forcing deploy it.